src/components/shared/MarkerTooltipContents.js

```diff
@@ -105,14 +148,50 @@ function getMarkerDetails(data: MarkerPayload): React$Element<*> | null {
                     timings.nonincremental_reason
                   )}
                 {_markerDetail(
+                  'gctime',
+                  'Total slice times',
+                  timings.total_time,
+                  x => x + 'ms'
+                )}
+                {_markerDetail(
                   'gcmaxpause',
                   'Max Pause',
                   timings.max_pause,
                   x => x + 'ms'
                 )}
+                {_markerDetail(
+                  'gcusage',
+                  'Heap usage',
+                  _formatBytes(timings.allocated)
+                )}
+                {_markerDetail(
+                  'gcmmu20ms',
+                  'MMU 20ms',
+                  timings.mmu_20ms,
+                  formatPercent
+                )}
+                {_markerDetail(
+                  'gcmmu50ms',
+                  'MMU 50ms',
+                  timings.mmu_50ms,
+                  formatPercent
+                )}
                 {_markerDetail('gcnumminors', 'Minor GCs', timings.minor_gcs)}
                 {_markerDetail('gcnumslices', 'Slices', timings.slices)}
-                {_markerDetail('gcnumzones', 'Zones', zones_collected_total)}
+                {_markerDetail(
+                  'gcnumzones',
+                  'Zones',
+                  _formatValueTotal(
+                    timings.zones_collected,
+                    timings.total_zones,
+                    String
```

Comment:
nit: maybe you can make `String` the default for this parameter, just like you did for `_markerDetail`